packages/neuron-wallet/tests/services/tx/transaction-persistor.test.ts

```diff
@@ -1,118 +1,11 @@
 import Transaction from "../../../src/models/chain/transaction"
-import CellDep, { DepType } from "../../../src/models/chain/cell-dep"
-import OutPoint from "../../../src/models/chain/out-point"
-import Input from "../../../src/models/chain/input"
-import Script, { ScriptHashType } from "../../../src/models/chain/script"
-import Output from "../../../src/models/chain/output"
 import { TransactionPersistor } from "../../../src/services/tx"
 import initConnection from "../../../src/database/chain/ormconfig"
 import TransactionEntity from "../../../src/database/chain/entities/transaction"
 import { getConnection } from "typeorm"
+import { loadTransactions } from '../../setupAndTeardown'
```

Comment:
Nice to see these test fixtures are located in one place.

Just one suggestion. Would it be more concise to name this file as `fixtures`, `setups` or `utils`, instead of `setupAndTeardown` since it doesn't seem to include the functions for teardowns?

Reply:
The method `closeConnection` in the `setupAndTeardown` clears the database so I'd like to keep the name of `setupAndTeardown`. But in this case, the transactions can be loaded directly from the fixture. I'll rename it.

Reply:
Updated in https://github.com/nervosnetwork/neuron/pull/1733/commits/b9b17e784713afb0d72ab7c7e19c24b97a61fc3b